clang/lib/Headers/avx512bitalgintrin.h

```diff
@@ -23,7 +23,7 @@
 static __inline__ __m512i __DEFAULT_FN_ATTRS
 _mm512_popcnt_epi16(__m512i __A)
 {
-  return (__m512i) __builtin_ia32_vpopcntw_512((__v32hi) __A);
+  return (__m512i)__builtin_elementwise_popcount((__v32hi)__A);
```

Comment:
hi or hu? The same below.

Reply:
I've updated all the x86 cases to use unsigned for consistency - `__builtin_elementwise_popcount` doesn't really care, although it expects src/dst types to match (although x86 casts back to _mXXXi types anyway).